test/logger.test.js

```diff
@@ -51,22 +49,87 @@ describe('strigo-node-logger', () => {
         log.debug(printout);
       });
 
-      expect(stdout.indexOf(printout)).to.be.eq(-1);
+      expect(stdout).to.be.empty;
     });
 
-    it('should allow setting up an express logger', () => {
-      const { logger: log, loggerMiddleware: middleware } = setupExpressLogger({ env: 'prod', level: 'info' });
-
-      // Should be a function. This test can be better if we check for specific
-      // expressWinston attributes.
-      expect({}.toString.call(middleware)).to.be.eq('[object Function]');
+    it('should output JSON structure', () => {
+      const log = setupNodeLogger({});
 
       const printout = chance.string();
       const stdout = capture.captureStdout(() => {
         log.info(printout);
       });
 
-      expect(stdout.indexOf(printout)).to.be.gt(-1);
+      expect(JSON.parse(stdout)).to.have.all.keys('@timestamp', 'log', 'message');
+      expect(JSON.parse(stdout)).to.deep.include({ log: { level: 'info' } });
+    });
+
+    it('should correctly place metadata', () => {
+      const log = setupNodeLogger({});
+
+      const msg = chance.string();
+      const workspace = chance.string();
+      const tag = chance.string();
+      const stdout = capture.captureStdout(() => {
+        log.info(msg, { workspaceId: workspace, tags: [tag] });
+      });
+
+      expect(JSON.parse(stdout)).to.deep.include({
+        message: msg, strigo: { workspaceId: workspace }, tags: [tag],
+      });
+    });
+
+    it('should handle errors according to ecs', () => {
+      const log = setupNodeLogger({});
+
+      const errmsg = chance.string();
+      const stdout = capture.captureStdout(() => {
+        log.info(new Error(errmsg));
+      });
+
+      expect(JSON.parse(stdout)).to.deep.include({ message: errmsg });
+      expect(JSON.parse(stdout)).to.have.nested.property('error.stack_trace');
     });
   });
+
+  describe('#verbose', () => {
+    it('should printout the configured severity level', () => {
+      const log = setupNodeLogger({ json: false });
+
+      const msg = chance.string({ symbols: false, alpha: true });
+      const stdout = capture.captureStdout(() => {
+        log.warn(msg);
+      });
+
+      // remember that the output has been colorized
+      const regex = new RegExp(`.+? - .+warn.+ - ${msg}`);
+      expect(stdout).to.match(regex);
+    });
+
+    it('should be multiline when metadata is added', () => {
```

Comment:
I don't think this describes it very well. Maybe `it('should output metadata in JSON format on a separate line')`?